cypress/pages/slices/slicePage.js

```diff
@@ -75,7 +75,7 @@ export class SlicePage {
    * @param {string} type Type of the widget.
    */
   addNewWidgetField(name, type) {
-    cy.contains("button", "Add a new Field").click();
```

Comment:
did this change?

Reply:
No, but for some reason addressing the button by text doesn't work when there is the second button no the screen (this happens when there are no fields in the slice)

Reply:
ah that makes sense